docs/source/development-environment.rst

```diff
@@ -0,0 +1,141 @@
+Zipline Development Guidelines
```

Comment:
Another thing that might be valuable here is a link to similar documents on larger, more established projects.

- The pandas docs have a good contribution guide here: http://pandas.pydata.org/pandas-docs/stable/contributing.html#
- GitHub has a good site for help docs: https://help.github.com/.